reference/providers/oracle.md

```diff
@@ -55,17 +55,16 @@ Lists to provide a virtual firewall for an instance.
 
 Deploys a group of VM instances in OCI.
 
-A new server group template is created for each new server group and stored in Oracle Object 
-Storage. The Oracle Cloud provider in Spinnaker creates a bucket named `_spinnaker_server_group_data`
-to store all the server group templates.
+The Oracle Cloud provider in Spinnaker creates a bucket named `_spinnaker_server_group_data`
+within the user's tenancy to keep track of all the server groups created.
```

Comment:
I notice you've removed mention of Oracle Object Storage. Are you assuming the reader (presumably an Oracle user) will know that's what you're referring to?

Reply:
yes, I changed it to Oracle Object Storage bucket for clarity.